sdk/cosmos/azure-cosmos-benchmark/ctl/run_benchmark.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+service_endpoint=$ctl_endpoint
+master_key=$ctl_key
+
+if [ -z "$ctl_operation" ]
+then
+operation=ReadLatency
+else
+operation=$ctl_operation
+fi
+
+if [ -z "$ctl_concurrency" ]
+then
+concurrency=50
+else
+concurrency=$ctl_concurrency
+fi
+
+if [ -z "$ctl_consistency_level" ]
+then
+consistency_level=Eventual
+else
+consistency_level=$ctl_consistency_level
+fi
+
+if [ -z "$ctl_number_of_operations" ]
+then
+number_of_operations=-1
+else
+number_of_operations=$ctl_number_of_operations
+fi
+
+if [ -z "$ctl_max_running_time_duration" ]
+then
+max_running_time_duration=PT10H
+else
+max_running_time_duration=$ctl_max_running_time_duration
+fi
+
+connection_mode=Direct
+number_of_precreated_documents=1000
```

Comment:
general idea on setting `number_of_precreated_documents` is that we should ensure on each BE partition at least there is a few document to read from (this would be a factor of Collection Throughput, partition splitting etc).

probably 1000 is good enough but to be safe can we set this to something higher (10,000) maybe?

Reply:
Done